test/tests/restart/pointer_restart_errors/tests

```diff
@@ -14,6 +14,8 @@
     type = 'RunApp'
     input = 'pointer_load_error.i'
     max_parallel = 1 # Sometimes times out in parallel after producing correct message
+    # this test is only used for generating the checkpoint files
+    recover = false
 
     requirement = 'The system shall store a custom type that contains a data serialization routine that is declared as restartable.'
```

Comment:
This test is only used for generating the checkpoint file. If we do not add `recover = false` for it, it runs in recover mode and now we will hit an error for a custom type that is missing `DataLoad` on purpose because we do not have early return in `init` as before.

Reply:
ok works for me